test/query_tests/try_commit_messages.test

```diff
@@ -0,0 +1,127 @@
+query: try_commit_messages
+
+debug: False
+
+mock_data:
+  header:
+    - user
+    - message
+  meta:
+    content_type: application/json
+    format: table
+  data:
+    - - wptsync@mozilla.com
+      - 
+    - - wptsync@mozilla.com
+      - 
+    - - gwatson@mozilla.com
+      - 
+    - - mh@glandium.org
+      - 
+    - - emilio@crisal.io
+      - 
+    - - dwillcoxon@mozilla.com
+      - 
+    - - pbone@mozilla.com
+      - 
+    - - gwatson@mozilla.com
+      - 
+    - - ktomlinson@mozilla.com
+      - 
+    - - dhouse@mozilla.com
```

Comment:
Could you invent some fake e-mail addresses for this query? You can use `example.com` as the domain name.

Reply:
Oh gosh! Didn't think of that. Will get right on it. Cheers.